### DISCUSSION TITLE
Please add an api to manually trigger the reveal (animate) the hiding/showing of the QHeader and QFooter

### DISCUSSION THREAD

**Discussion by brandon-wesley** [quasarframework/quasar#17357] [Ideas / Proposals]
I would like to use the reveal property of the header and footer but I am using a custom scroll setup which is not triggering the reveal mode. Can you please add an option for the animation of header when hiding it via the model property?

**Comment by ivanjaros** [comment]
i think you misunderstood what reveal does.

**Comment by brandon-wesley** [comment]
What makes you say that?

The reveal property simply animates the header & footer away on scroll, when used with the q-layout component. I am using another scroll setup and so I'm requesting that the animation be also applied to the manual updating of the model-value.

**Reply by ivanjaros** [reply]
> so I'm requesting that the animation be also applied to the manual updating of the model-value

header/footer reveal just uses https://quasar.dev/vue-directives/intersection with some "padding" via reveal-offset property. you cannot fake scrolling via v-model. i have no idea what you mean by "another scroll setup", but it sounds like you might maybe want to alter the layout's view property to change how header and footer are displayed during certain situation?
